libs/components/datetime/src/lib/modules/datepicker/calendar/datepicker-calendar-inner.component.scss

```diff
@@ -190,11 +193,13 @@
     }
 
     &:focus-visible:not(:active) {
+      // NOTE: The '0 0' as the fallback to the focus elevation is needed to get css to properly recognize a box shadow as a no-op here in the comma list.
+      // If we did not have the override then the whole shadow could be removed.
       box-shadow: var(
         --sky-override-button-box-shadow,
         inset 0 0 0 var(--sky-border-width-action-focus)
           var(--sky-color-border-action-tertiary-focus),
-        var(--sky-elevation-focus)
+        var(--sky-override-datepicker-button-box-shadow-focus-elevation, 0 0)
```

Comment:
what if instead of doing the `0 0` the whole value was an override for modern? i.e.

```
--sky-override-button-box-shadow: inset 0 0 0 var(--sky-border-width-action-focus) var(--sky-color-border-action-tertiary-focus), var(--sky-elevation-focus);
     
...

box-shadow: var(--sky-override-button-box-shadow, inset 0 0 0 var(--sky-border-width-action-focus) var(--sky-color-border-action-tertiary-focus));
```

Reply:
Done